doc/quickstart.md

```diff
@@ -22,7 +22,7 @@ Here's how to create a dashboard:
 ```js
 cartodb.deepInsights.createDashboard('#dashboard', vizJSONurl, {}, function(err, dashboard) {
     // update a dashboard widget
-    var formulaWidget = dashboard.getWidget('whatever-id')
+    var formulaWidget = dashboard.widgets.get('whatever-id')
```

Comment:
not agree with this, let's keep it simple, having objects to access is harder than having a method.